tests/apikey_test.go

```diff
@@ -0,0 +1,14 @@
+package easypost_test
+
+func (c *ClientTests) TestUserApiKeysAll() {
```

Comment:
Re-record this cassette please so it picks up the new name.

```suggestion
func (c *ClientTests) TestApiKeysAll() {
```